src/main/java/com/meronat/latch/commands/Commands.java

```diff
@@ -212,6 +216,36 @@ public void register() {
                         GenericArguments.optionalWeak(GenericArguments.string(Text.of("password"))))
                 .build();
 
+        final CommandSpec addLockableCommand = CommandSpec.builder()
+                .description(Text.of("Add a block type to the lockable block list"))
+                .permission("latch.admin.lockable.add")
+                .executor(new AddLockableCommand())
+                .arguments(GenericArguments.optionalWeak(
+                        GenericArguments.onlyOne(GenericArguments.catalogedElement(Text.of("entered"), BlockType.class))))
+                .build();
+
+        final CommandSpec removeLockableCommand = CommandSpec.builder()
+                .description(Text.of("Remove a block type from the lockable block list"))
+                .permission("latch.admin.lockable.remove")
+                .executor(new RemoveLockableCommand())
+                .arguments(GenericArguments.optionalWeak(
+                        GenericArguments.onlyOne(GenericArguments.catalogedElement(Text.of("entered"), BlockType.class))))
+                .build();
+
+        final CommandSpec listLockableCommand = CommandSpec.builder()
+                .description(Text.of("List the block types on the lockable block list"))
+                .permission("latch.admin.lockable.list")
+                .executor(new ListLockableCommand())
+                .build();
+
+        final CommandSpec lockableCommand = CommandSpec.builder()
+                .description(Text.of("Base command for modifying lockable block list"))
```

Comment:
Could be 'modifying the lockable' for consistency with the others.

Reply:
Sure